.drone.star

```diff
@@ -1599,7 +1599,7 @@ def makeNodeGenerate(module):
                 "CHROMEDRIVER_SKIP_DOWNLOAD": "true",  # install fails on arm and chromedriver is a test only dependency
             },
             "commands": [
-                "%s ci-node-generate" % (make),
+                "retry -t 3 '%s ci-node-generate'" % (make),
```

Comment:
```suggestion
                "/usr/local/bin/retry -t 3 '%s ci-node-generate'" % (make),
```

Maybe the PATH list in these images does not look in `/usr/local/bin` - maybe we have to give the full path?